lib/src/driver.dart

```diff
@@ -59,15 +61,27 @@ class Driver {
   /// Use the given command-line [args] to start this analysis driver.
   void start(List<String> args) {
     StringUtilities.INTERNER = new MappedInterner();
+
     _processPlugins();
+
+    // Parse commandline options.
     CommandLineOptions options = CommandLineOptions.parse(args);
-    // In batch mode, SDK is specified on the main command line rather than in
-    // the command lines sent to stdin.  So process it before deciding whether
-    // to activate batch mode.
-    if (sdk == null) {
-      sdk = new DirectoryBasedDartSdk(new JavaFile(options.dartSdkPath));
+
+    // Set up env.
+    {
```

Comment:
I don't understand the value of the added block, but I guess I don't care if you leave it.